extensions/arc/src/models/postgresModel.ts

```diff
@@ -93,8 +95,19 @@ export class PostgresModel extends ResourceModel {
 			configuration.push(`${ramLimit ?? ramRequest!} ${loc.ram}`);
 		}
 
-		if (storage) {
-			configuration.push(`${storage} ${loc.storagePerNode}`);
+		let storage: string[] = [];
+		if (dataStorage) {
+			storage.push(` ${dataStorage} data`);
```

Comment:
These strings should be localized. And make sure you do it by passing in the parameter so the strings can be ordered correctly based on language syntax